pkg/controller/v1beta1/inferenceservice/rawkube_controller_test.go

```diff
@@ -576,7 +577,7 @@ var _ = Describe("v1beta1 inference service controller", func() {
 				if err := k8sClient.Get(context.TODO(), serviceKey, isvc); err != nil {
 					return err.Error()
 				}
-				return cmp.Diff(&expectedIsvcStatus, &isvc.Status, cmpopts.IgnoreTypes(apis.VolatileTime{}))
+				return cmp.Diff(&expectedIsvcStatus, &isvc.Status, cmpopts.IgnoreTypes(apis.VolatileTime{}), cmpopts.IgnoreFields(v1beta1.InferenceServiceStatus{}, "DeploymentMode"))
```

Comment:
Why do we need to ignore the deploymentMode field? it should be in the expected isvc status

Reply:
I've fixed in the replacement PR #4423